modules/troubleshooting-disabling-autoreboot-mco-cli.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco-cli_{context}"]
+= Disabling Machine Config Operator from automatically rebooting by using the CLI
+
+To avoid unwanted disruptions from changes made by the Machine Config Operator (MCO), you can use the CLI to modify the machine config pool (MCP) to prevent automatic rebooting after the Operator makes changes to the machine config.
+
+[NOTE]
+====
+Pausing a machine config pool stops all system reboot processes and all configuration changes from being applied.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have root access in {product-title}.
+
+.Procedure
+. To pause the autoreboot process after machine config changes are applied:
+
+* As root, update the `spec.paused` field to `true` in the `MachineConfigPool` custom resource.
++
+.Control plane (master) nodes
+[source,terminal]
+----
+# oc patch --type=merge --patch='{"spec":{"paused":true}}' machineconfigpool/master
+----
++
+.Worker nodes
+[source,terminal]
+----
+# oc patch --type=merge --patch='{"spec":{"paused":true}}' machineconfigpool/worker
+----
+
+. To verify that the machine config pool is paused:
++
+.Control plane (master) nodes
+[source,terminal]
+----
+# oc get machineconfigpool/master --template='{{.spec.paused}}'
+----
++
+.Worker nodes
+[source,terminal]
+----
+# oc get machineconfigpool/worker --template='{{.spec.paused}}'
+----
++
+The `spec.paused` field is `true` and the machine config pool is paused.
+
+. Alternatively, to unpause the autoreboot process:
+
+* As root, update the `spec.paused` field to `false` in the MachineConfigPool CustomResourceDefinition (CRD).
```

Comment:
Hmm, should it be `CustomResource` perhaps? We're not modifying the definition, just the resource itself